src/components/Link/Link.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <h5>
+    <a class="Link"
```

Comment:
Since we will be using this link in the context of nuxt it'd better you use the [nuxt-link](https://nuxtjs.org/api/components-nuxt-link) instead, so we are able to take advantage of nuxt routing. :)